Tests/OrderedCollectionsTests/OrderedDictionary/OrderedDictionary+Values Tests.swift

```diff
@@ -24,6 +57,16 @@ class OrderedDictionaryValueTests: CollectionTestCase {
     ]
     expectEqualElements(d.values, [1, 2, 3, 4])
   }
+  
+  func test_values_getter_not_equal_elements() {
+    let d: OrderedDictionary = [
+      "one": 1,
+      "two": 2,
+      "three": 3,
+      "four": 4,
+    ]
+    expectNotEqualElements(d.values, [1, 2, 3, 4, 5])
+  }
```

Comment:
On second look, this test seems to be mostly testing the standard `elementsEqual` algorithm; it doesn't add anything that's specific to `OrderedDictionary.Values` but that isn't already covered by the `checkBidirectionalCollection(d.values, expectedContents: values)` test in an earlier test method above.

That protocol validation test verifies that the sequence has the expected contents (in addition to a bunch of other things), which is all we need! We do not need a specific test to verify that the contents do not also match some other list -- that can be safely assumed to be the case. 

(Such a check may be useful if we were testing `elementsEqual` itself, but this isn't the right place for that.)

```suggestion
```